### ISSUE TITLE
can you give me your username please

### ISSUE THREAD

**Issue by amqndin** [Amandin/project#1]
it seems as though you dont even use github

**Comment by Sleet827** [comment]
yes I also think you should give amqndin their username because your last contribution was 12 years ago so it probably doesn't matter and amandin's name everywhere on social media is amandin, except github, where it's amqndin

**Comment by Sleet827** [comment]
fr
